pkg/splunk/enterprise/monitoringconsole.go

```diff
@@ -127,7 +127,6 @@ func ApplyMonitoringConsole(client splcommon.ControllerClient, cr *enterpriseApi
 		} else {
 			result.Requeue = false
 		}
-		result.Requeue = false
```

Comment:
Just confirming, is this an expected change?

Reply:
yes, this is expected. Condition is fulfilled in else block line #128